src/pages/en/core-concepts/project-structure.md

```diff
@@ -101,6 +101,8 @@ As a general rule, any CSS or JavaScript that you write yourself should live in
 
 This is a file used by JavaScript package managers to manage your dependencies. It also defines the scripts that are commonly used to run Astro (ex: `npm start`, `npm run build`).
 
+There are two kinds of dependencies you can specify in a `package.json`: `dependencies` and `devDependencies` ([see the NPM docs](https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file)). In most cases, these work the same: Astro needs all dependencies at build time, and your package manager will install both. We recommend putting all of your dependencies in `dependencies` to start.
```

Comment:
```suggestion
There are [two kinds of dependencies](https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file) you can specify in a `package.json`: `dependencies` and `devDependencies`. In most cases, these work the same: Astro needs all dependencies at build time, and your package manager will install both. We recommend putting all of your dependencies in `dependencies` to start, and only use `devDependencies` if you find a specific need to do so.
```
What do you think about moving the reference link here?

Reply:
I like this, reads better